packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason-form.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { Formik, Field } from 'formik';
+import { Checkbox, Input, FormSubmitButton, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { PlatformContext } from '@deriv/shared';
+
+const initial_form = {
+    'I have other financial priorities': false,
+    'I want to stop myself from trading': false,
+    "I'm no longer interested in trading": false,
+    'I prefer another trading website': false,
+    "The platforms aren't user-friendly": false,
+    'Making deposits and withdrawals is difficult': false,
+    'The platforms lack key features or functionality': false,
+    'Customer service was unsatisfactory': false,
+    "I'm deactivating my account for other reasons": false,
+    other_trading_platforms: '',
+    do_to_improve: '',
+};
+
+const DeactivateAccountReasonFrom = ({
+    validateFields,
+    onSubmit,
+    is_checkbox_disabled,
+    onChangeCheckbox,
+    character_limit_no,
+    onInputChange,
+    onInputPaste,
+    remaining_characters,
+    onBackClick,
+}) => {
+    const { is_dashboard } = React.useContext(PlatformContext);
+
+    return (
+        <Formik initialValues={initial_form} validate={validateFields} onSubmit={onSubmit}>
+            {({ values, setFieldValue, errors, handleChange, handleSubmit }) => (
+                <form onSubmit={handleSubmit}>
+                    <Field name='I have other financial priorities'>
```

Comment:
I believe this has to be localized as well?

Reply:
Or if it's not to be shown to users, maybe we can use short names instead of the full sentences

Reply:
since it's just name we not need localize for that, But you're right, we can use short names instead of sentences